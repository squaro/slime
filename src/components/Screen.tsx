import styled from 'styled-components';

const Screen = styled.section`
  position: relative;
  width: 100%;
  height: 100%;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  z-index: 0;
`;

export default Screen;
