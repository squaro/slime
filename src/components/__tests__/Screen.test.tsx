import { render } from '@testing-library/react';
import Screen from '../Screen';

describe('Screen', (): void => {
  it('renders the screen component', (): void => {
    // Act
    render(<Screen />);

    // Assert
    expect(document.body).toMatchSnapshot();
  });
});
