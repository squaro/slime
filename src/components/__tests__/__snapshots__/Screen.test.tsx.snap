// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Screen renders the screen component 1`] = `
<body>
  <div>
    <section
      class="sc-bdfBwQ lghxSv"
    />
  </div>
</body>
`;
